### PR TITLE
Ce/ben reg backport

### PR DESCRIPTION
**Pull request by calellowitz**
@proteusvacuum 
after a long discussion with the enikshay reports team, it looks like the data source i wanted to use for the updated version of this report is not yet tested and does not support the existing report. they also mentioned it would be best to make changes to the live beneficiary register as opposed to a new report (as i believe @proteusvacuum suggested as well). I did some qa on v2 today and after fixing some bugs the adherence score, this PR backports the data source and report changes to the report and data source. it goes a bit on a tangent where i was going to create a new data source, but i think this is better. the only report that uses the one listed is this report and i am confident the data source can rebuild fully overnight.

### CODE REVIEW

File: custom/enikshay/ucr/data_sources/episode_v3_1.json [hunk 1]
```diff
@@ -16,22 +16,20 @@
...
-                    "expression": {
-                        "datatype": null,
-                        "type": "property_name",
-                        "property_name": "type"
-                    },
-                    "type": "boolean_expression",
-                    "property_value": "episode"
-                }
-            ]
+            "filters": {
```
Comment (esoergel) [inline custom/enikshay/ucr/data_sources/episode_v3_1.json]: Flagging that you changed `filters` from a list to a dict, but this is an `and` filter.  Does that work?

File: custom/enikshay/ucr/data_sources/episode_v3.json [hunk 2]
```diff
@@ -6,8 +6,7 @@
         "enikshay-performance-test"
     ],
     "server_environment": [
-        "enikshay",
-        "softlayer"
```
Comment (esoergel) [inline custom/enikshay/ucr/data_sources/episode_v3.json]: Nice catch

File: custom/enikshay/ucr/data_sources/episode_v3.json [hunk 3]
```diff
@@ -18,17 +18,14 @@
...
-                "property_value": "episode"
-            }
+            "operator": "eq",
+            "expression": {
+                "datatype": null,
+                "type": "property_name",
+                "property_name": "type"
+            },
+            "type": "boolean_expression",
+            "property_value": "episode"
```
Comment (esoergel) [inline custom/enikshay/ucr/data_sources/episode_v3.json]: 👍

**Review by esoergel — APPROVED** [review]
Seemed reasonable enough.  You're sure this is best done in-place rather than as a separate report?

File: custom/enikshay/ucr/data_sources/episode_v3_1.json (hunk 1 above)
Comment (esoergel) [reply custom/enikshay/ucr/data_sources/episode_v3_1.json]: (saw that you resolved this later)

File: custom/enikshay/ucr/reports/beneficiary_register.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
         "enikshay",
         "softlayer"
     ],
-    "data_source_table": "episode_v3",
+    "data_source_table": "episode_v3_1",
```
Comment (proteusvacuum) [inline custom/enikshay/ucr/reports/beneficiary_register.json]: where is this defined?

**Pull request merged**